robot/pmm/tests/browser/ProgramCohort/Create_Program_Cohort.robot

```diff
@@ -15,29 +15,73 @@ Setup Test Data
     ${ns} =                 Get PMM Namespace Prefix
     Set suite variable      ${ns}
     ${program_cohort} =     Generate Random String
-    Set suite variable      ${program_cohort}
-    &{program} =            API Create Program
-    Store Session Record    ${ns}Program__c         &{program}[Id]
-    Set suite variable      &{program}
+    Set suite variable      ${program_cohort}      
+    &{program} =                API Create Program  
+    Store Session Record        ${ns}Program__c         ${program}[Id]
+    Set suite variable          &{program}
 
 
 *** Test Cases ***
-
-Create Program Cohort from Program Object
+Create Program Cohort
+     [Documentation]                        This test creates a program cohort and verifies the details of the page
```

Comment:
can u expand on what details are verified on the page

Reply:
updated the documentation and also fixed cases for keywords